test/serviceprofiles/serviceprofiles_test.go

```diff
@@ -53,23 +70,23 @@ func TestServiceProfiles(t *testing.T) {
 		{
 			sourceName:     "tap",
 			namespace:      testNamespace,
-			deployName:     "deploy/t1",
+			deployName:     "deployment/t1",
```

Comment:
curious: was this change necessary to match against returned data from a routes command?

Reply:
Yea, I had to use the full `deployment` name in order to get stuff back. I can make a change to the routes command to accept both `deploy` and `deployment`